Modix/Modules/CatModule.cs

```diff
@@ -0,0 +1,178 @@
+﻿namespace Modix.Modules
+{
+    using System.Net.Http;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Discord.Commands;
+    using Newtonsoft.Json;
+    using Serilog;
+
+    public class CatModule : ModuleBase
+    {
+        // Url for cat
+        public struct URL
+        {
+            public string file { get; set; }
+        }
+
+        private static readonly HttpClient Client = new HttpClient();
+
+        [Command("cat"), Summary("Gets a cat")]
+        public async Task Cat(string gif = "")
+        {
+            // 30 seconds to find a cat
+            var cts = new CancellationTokenSource(30000);
```

Comment:
This can be much shorter, probably 5 or so seconds at most. For comparison, the eval module waits 15 seconds.